prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -240,7 +241,11 @@ def get_storage_controller_data(self) -> Tuple[Dict[str, int], Dict[str, List]]:
         storage_controller_data: Dict[str, List] = {}
         # storage controllers on each system
         storage_controller_count: Dict[str, int] = {}
-        storage_root_uri_pattern = "/redfish/v1/Systems/{}/Storage"
+        systems_root_uri = "/redfish/v1/Systems/"
+
+        # storage_root_uri becomes systems_root_uri/system_id/storage_name
+        # eg: "/redfish/v1/Systems/S1/Storage"
+        storage_root_uri_pattern = systems_root_uri + "{}/{}"
```

Comment:
Would not be better to use pathlib instead of predefined string? Or using simple function to generate url?
e.g.:
```python
def _storage_uri(self, system_id, storage_name):
    return self.systems_root_uri + f"{system_id}/{storage_name}"
```